src/GitLink/Helpers/ProjectHelper.cs

```diff
@@ -17,37 +18,19 @@ namespace GitLink
     using Microsoft.Build.Evaluation;
     using System.IO;
     using System.Text.RegularExpressions;
+    using ImpromptuInterface;
+    using Interfaces;
```

Comment:
No namespace for interfaces, just a folder so they are all together. Same for .Helpers, .Extensions, etc. A namespace should represent a functionality group, not a folder on disk.